### PR TITLE
Update signing-in.md with troubleshooting TOC bullet point

### PR DESCRIPTION
**Pull request by leslierichardson95**

### CODE REVIEW

File: docs/csharp/signing-in.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ In this article, you'll learn:
 * How to [sign in](#sign-in-with-a-microsoft-or-organizational-account) with an account.
 * How to check the [status of your Visual Studio subscription](#verifying-your-visual-studio-subscription-status).
 * How to [sign out of your account](#how-to-sign-out).
+* * How to [troubleshoot sign in issues](#troubleshoot-sign-in-issues)
```
Comment (gregvanl) [inline docs/csharp/signing-in.md:16]: Looks like extra *?

Comment (gregvanl) [reply docs/csharp/signing-in.md]: fixed

File: docs/csharp/signing-in.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ In this article, you'll learn:
 * How to [sign in](#sign-in-with-a-microsoft-or-organizational-account) with an account.
 * How to check the [status of your Visual Studio subscription](#verifying-your-visual-studio-subscription-status).
 * How to [sign out of your account](#how-to-sign-out).
+* How to [troubleshoot sign in issues](#troubleshoot-sign-in-issues)
```
Comment (cadamini) [inline docs/csharp/signing-in.md:16]:
```suggestion
* How to [troubleshoot sign in issues](#troubleshoot-sign-in-issues).
```

You missed the period. The rest of the list has periods.

**Review by cadamini — APPROVED** [review]
LGTM, see my comment.

**Review by gregvanl — APPROVED** [review]

**Review by deepak1556 — APPROVED** [review]

**Comment by gregvanl** [comment]
@leslierichardson95 Published

**Comment by leslierichardson95** [comment]
> @leslierichardson95 Published

Thanks for publishing and fixing my typo :)

**Pull request merged**